src/menu/rendersettingsmenustate.cpp

```diff
@@ -352,16 +352,28 @@ RenderSettingsMenuState::RenderSettingsMenuState(const std::shared_ptr<MenuRingT
     {
       toggle(engine, engine.getEngineConfig()->renderSettings.fullscreen);
     });
+ 
+   m_dustDensitySelector = std::make_shared<ui::widgets::ValueSelector<float>>(
+    [](float value)
+    {
+      return /* translators: TR charmap encoding */ _("\x1f\x6c %1% \x1f\x6d Dust Particles",
+                                                      static_cast<float>(value));
+    },
+    std::vector<float>{100.0f, 50.0f, 33.33f, 25.0, 20.0f, 16.66f, 14.28f, 12.5f});
```

Comment:
The `1/n` approach was more clear, because it expressed the intention of the math. I would have changed only the representation, i.e. `_("\x1f\x6c %1%%% \x1f\x6d Dust Particles", 100/value)`

Reply:
I don't seem to be able to draw a % sign in the TR font. It renders as a strange looking B. I also tried \x19 as that's the character code according to windows character map, but that renders as a blank space.

Reply:
Ah, sorry, seems the percent sign is not part of the original TR typeface. How about `"\x1f\x6c %1% \x1f\x6d Percent Dust Particles"`?